Specifications/FeelBot1.feature

```diff
@@ -0,0 +1,6 @@
+Scenario : FeelBotCore1
```

Comment:
I feel that there should be already a naming convention on the features and feature files... Could you implement one and document it in the wiki ? 
Tkx :)